fsx/README.md

```diff
@@ -0,0 +1,41 @@
+# miniwdl-aws-terraform/fsx
+
+This variant of the [miniwdl-aws-terraform](https://github.com/miniwdl-ext/miniwdl-aws-terraform) configuration deploys [FSx for Lustre](https://aws.amazon.com/fsx/lustre/) (FSxL) instead of [EFS](https://aws.amazon.com/efs/). FSxL has greater upfront costs than EFS, but offers higher throughput scalability needed for large-scale operations.
+
+Compared to the default EFS stack, the key differences here are:
+
+1. FSxL and Batch compute environments are confined to one availability zone
```

Comment:
Then you gonna incur cross-AZ charges for filesystem traffic, right? (whereas EFS lets you create a mount target in each AZ)

Reply:
Yes, there will be cross AZ data xfer costs. That said, in practice they will be much smaller than the compute costs for tasks in a workflow.

Reply:
In addition, if the FSxL has a public IP and the Batch workers have private IPs (even in the same AZ) then I believe you will incur NAT gateway charges. This would be another reason to keep both private.